node/src/main/scala/com/wavesplatform/Exporter.scala

```diff
@@ -97,9 +96,8 @@ object Exporter extends ScorexLogging {
         .getOrElse(0)
     }
 
-    def exportBlockToJson(stream: OutputStream, db: DB, height: Int): Int = {
-      val maybeBlock = db.readOnly(ro => database.loadBlock(Height(height), ro))
-      maybeBlock
+    def exportBlockToJson(stream: OutputStream, maybeBlock: () => Option[Block], height: Int): Int = {
```

Comment:
It seems, we don't need `maybeBlock` to be a function here, just `Option[Block]` is enough, because we call it on the first line